packages/connect-web-bench/README.md

```diff
@@ -10,5 +10,5 @@ it like a web server would usually do.
 
 | code generator | bundle size        | minified               | compressed           |
 |----------------|-------------------:|-----------------------:|---------------------:|
-| connect        | 117,734 b | 51,217 b | 13,697 b |
+| connect        | 122,099 b | 53,623 b | 14,402 b |
```

Comment:
This could be caused by new version of esbuild. We have to update esbuild separately, so that we have a good understanding of bundle size changes.

We'll update deps tomorrow - hopefully this (and changes from prettier) will automatically resolve.